config/knative-cluster-issuer.yaml

```diff
@@ -21,34 +21,37 @@ metadata:
     app.kubernetes.io/version: devel
     app.kubernetes.io/name: knative-serving
     networking.knative.dev/certificate-provider: cert-manager
+    knative.dev/issuer-install: "true"
```

Comment:
Could you explain where we use that?

Reply:
Nowhere yet. My idea was that people do not want to install this in production, so you can filter on that (e.g. we probably want do to this in the downstream operator). I'll add it to the install docs accordingly.